src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,52 +1,112 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
 public class ArrayList<T> implements List<T> {
 
+    private static final int DEFAULT_CAPACITY = 10;
+
+    private Object[] data;
+    private int currentCapacity;
+    private int pointer;
```

Comment:
@kozhukhovsky it was better to name this variable size

Reply:
ok, thx